WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -441,7 +441,7 @@ private async Task FailTransactionSigningPhaseAsync(Round round, CancellationTok
 
 		round.LogInfo($"Removed {cnt} alices, because they didn't sign. Remaining: {round.InputCount}");
 
-		if (round.InputCount >= round.Parameters.MinInputCountByRound)
+		if (round.InputCount >= Config.MinInputCountByBlameRound)
```

Comment:
I can't wrap my head around this change. 
Why is it necessary?

We have lots of checks where `round.InputCount` is checked against `round.Parameters.MinInputCountByRound`, but you only modified it here.

Am I missing something?

Reply:
> on this PR, the min input count for a blame round will be different than the round that failed, it will be
Config.MinInputCountByBlameRound, so I have to check against this.

omg, you are so right.

> So this line would look like: if (round.InputCount >= round.Parameters.MinInputCountByRound * Config.MultiplierMinInputCountBlameRound)

IDK, which one the team prefers. 
For me, the current solution is fine. Multiplying inside the `if` can be a bit hard to understand at first glance.

Don't forget to change this line, it's about creating blame rounds as well.
https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs#L469-L473